code/game/gamemodes/abduction/abduction_gear.dm

```diff
@@ -557,4 +557,56 @@ Congratulations! You are now trained for xenobiology research!"}
 	name = "alien locker"
 	desc = "Contains secrets of the universe"
 	icon_state = "abductor"
-	icon_door = "abductor"
\ No newline at end of file
+	icon_door = "abductor"
+
+
+
+//Abductor augs
+
+/obj/item/robot_parts/abductor
+	name = "aliem robotic part"
```

Comment:
yeah, not worth fixing the typo to be honest, if I have to commit something else I'll fix it, but it's unseen text, it's fine.